modules/stanford_image_styles_preview/src/Form/PreviewForm.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Drupal\stanford_image_styles_preview\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Class PreviewForm.
+ */
+class PreviewForm extends FormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'preview_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = [];
+
+    $form['image'] = [
+      '#type' => 'managed_file',
```

Comment:
I tried a normal file but with a required field it wouldn't validate. I wasn't sure why. I can try to see if theres a work around